modules/unsupported/cql-json/src/main/java/org/geotools/filter/text/cqljson/CQL2Json.java

```diff
@@ -0,0 +1,65 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2022, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.filter.text.cqljson;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.geotools.filter.FilterFactoryImpl;
+import org.geotools.filter.text.cql2.CQLException;
+import org.opengis.filter.Filter;
+
+/**
+ * <b>OGC API CQL2-Json</b>. This class presents the operations available to convert filters into
+ * the CQL2-Json language
+ *
+ * <p>
+ */
+public class CQL2Json {
```

Comment:
I'd add static methods handling Expression as well, mirroring CQL2, CQL and ECQL.

Reply:
Added